epub33/core/index.html

```diff
@@ -657,15 +657,37 @@ <h4>Introduction</h4>
 						without fallbacks (<a>Core Media Type Resources</a>) and those that cannot (<a>Foreign
 							Resources</a>).</p>
 
-					<p><a>Authors</a> are free to use both types of resources to construct their EPUB Publications, but
-						need to be aware that some Reading Systems might not render the Foreign Resources they use.</p>
-
-					<p>As EPUB Publications are designed to be fully consumable on any compliant Reading System, a
-						system of fallbacks is necessary to ensure that the use of Foreign Resources does not impact on
-						the ability of the user to consume the content. This section lists the <a
-							href="#sec-cmt-supported">set of Core Media Type Resources</a> and identifies <a
-							href="#sec-foreign-restrictions">fallback mechanisms</a> that can be used to satisfy the
-						consumability requirement.</p>
+					<p>Formats are typically only included as Core Media Type Resources when it can be shown that they
+						have broad support in web browser cores &#8212; the rendering engines on which EPUB 3 Reading
+						Systems are built. They are like a compact between <a>Authors</a> and Reading Systems to ensure
```

Comment:
I think we may have to discuss more about what we mean by "broad support in web browser cores". As it stands, this is fairly vague, making it difficult to judge requests like #1344.

Reply:
I'm fine with being more specific, but it's not something we've discussed (it would also likely be normative, which would have to be specified outside the intro). Feel free to open a separate issue for it.

Reply:
I actually believe that @wareid may put it on one of our meeting agendas... 

So you are right, this is a discussion for another day.